tracing/src/lib.rs

```diff
@@ -155,9 +155,9 @@
 //! fields using `fmt::Debug`.
 //!
 //! For example:
-//! ```ignore
-//! # // this doctest is ignored because we don't have a way to say
-//! # // that it should only be run with cfg(feature = "attributes")
+//! ```
```

Comment:
Please add ` ```rust ` here, so that we stay consistent :)

Reply:
Done. I've added it everywhere in this file, but for the record, there are still some places where it is not used. I originally decided against changing it explicitly everywhere so that there are not unnecessary changes.

Arguably the one you pointed to was already a change but I thought I'd just go with whatever was at the place.